### PR TITLE
Add label tag

### PR DESCRIPTION
**Pull request by hardik-codes**
Pull Request for Issue #24467 

### Summary of Changes
Add ```label``` tag to search field

### Testing Instructions
Code Inspection

### Documentation Changes Required
None

### CODE REVIEW

**Comment by hardik-codes** [comment]
@zwiastunsw please test

**Comment by tahseen-fathima98** [comment]
Where can i see the code and try to do the changes in it?
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24482">issues.joomla.org/tracker/joomla-cms/24482</a>.</sub>

**Comment by ghost** [comment]
I guess you mean doing a Test so please read https://brian.teeman.net/joomla/873-a-dummies-guide-to-joomla-bug-testing or https://docs.joomla.org/Testing_Joomla!_patches

File: components/com_search/views/search/tmpl/default_form.php [hunk 1]
```diff
@@ -18,6 +18,9 @@
 <form id="searchForm" action="<?php echo JRoute::_('index.php?option=com_search'); ?>" method="post">
 	<div class="btn-toolbar">
 		<div class="btn-group pull-left">
+			<label for="search-searchword" class="element-invisible">
+			<?php echo JText::_('COM_SEARCH_SEARCH_KEYWORD'); ?>
```
Comment (Quy) [inline components/com_search/views/search/tmpl/default_form.php]:
```suggestion
				<?php echo JText::_('COM_SEARCH_SEARCH_KEYWORD'); ?>
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f44b2b12f746bf002726eed357fabbad229bd4fc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24482">issues.joomla.org/tracker/joomla-cms/24482</a>.</sub>

**Comment by hardik-codes** [comment]
one more tester please

**Comment by brianteeman** [comment]
@hardik-codes please dont keep requesting tests. When people have the time they will test.

**Comment by HLeithner** [comment]
@hardik-codes you can join the joomla bug squad and talk to people or ask students of the gsoc project to test PRs in the gsoc glip chat. Ping me on glip.

**Comment by hardik-codes** [comment]
> @hardik-codes please dont keep requesting tests. When people have the time they will test.

I'm really sorry about that. Will keep this in mind from next time onwards

**Comment by hardik-codes** [comment]
> @hardik-codes you can join the joomla bug squad and talk to people or ask students of the gsoc project to test PRs in the gsoc glip chat. Ping me on glip.

sure

**Comment by zwiastunsw** [comment]
I have tested this item :white_check_mark: successfully on f44b2b12f746bf002726eed357fabbad229bd4fc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24482">issues.joomla.org/tracker/joomla-cms/24482</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by SniperSister** [comment]
Merged! Thanks a lot :)

**Pull request merged**